nuxeo-distribution/nuxeo-server-tests/itests.xml

```diff
@@ -16,4 +16,22 @@
     <artifact:resolveFile key="org.nuxeo:nuxeo-ftest::zip" />
   </unzip>
   <import file="${out.dir}/nuxeo-ftest.xml" />
+
+  <target name="prepare-environment" depends="_init,prepare-db,prepare-tomcat">
+    <!-- enable dev mode, needed by dev hot reload tests -->
+    <antcall target="set-conf">
+      <param name="name" value="org.nuxeo.dev" />
+      <param name="value" value="true" />
+    </antcall>
```

Comment:
not sure we want dev mode to be activated for all server tests, maybe this hotreload test should be moved to a dedicated module that sets up a server in dev mode (?)